src/webgl/p5.Framebuffer.js

```diff
@@ -842,9 +842,10 @@ class Framebuffer {
     // RendererGL.reset() does, but this does not try to clear any buffers;
     // it only sets the camera.
     this.target.setCamera(this.defaultCamera);
-    this.target._renderer.uMVMatrix.set(
-      this.target._renderer._curCamera.cameraMatrix
-    );
+    this.target.resetMatrix();
+    this.target._renderer.uViewMatrix
+      .set(this.target._renderer._curCamera.cameraMatrix);
+    this.target._renderer.uModelMatrix.set(this.target.modelMatrix);
```

Comment:
I think the target context doesn't have a `modelMatrix` property, so after setting the view matrix, we just need to reset the model matrix:
```suggestion
    this.target._renderer.uModelMatrix.reset();
```